int16 setpoint_x
int16 state_x
int16 setpoint_y
int16 state_y
